autogluon/searcher/bayesopt/autogluon/openml_download.py

```diff
@@ -0,0 +1,60 @@
+import os
```

Comment:
Should this file live somewhere else? It doesn't seem to be particular to bayesopt. Maybe in autogluon/utils?

Reply:
It is some pretty specific piece of code that was previously in Thibaut's tutorial, and is now also used in a unit test. Do you think it would be more generally useful?

I could also use a file collecting other helpers for unit tests of bayesopt, and put it there.

Reply:
I would agree with nick's suggestion, put the download utils somewhere in utils is better IMO